src/components/parameters/ValueSourceBadge.svelte

```diff
@@ -1,21 +1,56 @@
 <svelte:options immutable={true} />
 
 <script lang="ts">
+  import { createEventDispatcher } from 'svelte';
   import type { ValueSource } from '../../types/parameter';
-  import { classNames } from '../../utilities/generic';
+  import { classNames, isMacOs } from '../../utilities/generic';
+  import { isMetaOrCtrlPressed } from '../../utilities/keyboardEvents';
   import { tooltip } from '../../utilities/tooltip';
 
   export let source: ValueSource;
 
+  const dispatch = createEventDispatcher();
+
+  let tooltipContent: string = '';
+
   $: dotClasses = classNames('value-source-badge-dot st-typography-body', {
     'value-source-badge-dot--mission': source === 'mission',
-    'value-source-badge-dot--user': source === 'user',
+    'value-source-badge-dot--preset': source === 'preset',
+    'value-source-badge-dot--user': source === 'user on model' || source === 'user on preset',
   });
-  $: tooltipContent = source === 'mission' ? 'Default Mission Value' : source === 'user' ? 'User Override' : '';
+  $: switch (source) {
+    case 'user on model':
+    case 'user on preset':
+      tooltipContent = `<div class="value-source-tooltip-modified">
+                          <span>Modified</span>
```

Comment:
Can we remove all the extra whitespace here?

Reply:
You'd rather have this all one one line?